students/JaeKim/lesson07/assignment/linear.py

```diff
@@ -0,0 +1,105 @@
+import os
+import csv
+from pymongo import MongoClient
+
+
+class MongoDBConnection():
+    """MongoDB Connection"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
+
+
+def cleanup():
+    mongo = MongoDBConnection()
+
+    with mongo:
+        database = mongo.connection.media
+
+        [i.drop() for i in [database.products, database.customers, database.rentals]]
```

Comment:
I like the comprehension, however you shouldn't make a list if you aren't using it; just uses up extra space. Just drop in a regular loop.